Yosemite/Yosemite/Actions/OrderAction.swift

```diff
@@ -65,6 +65,10 @@ public enum OrderAction: Action {
     ///
     case updateOrder(siteID: Int64, order: Order, fields: [OrderUpdateField], onCompletion: (Result<Order, Error>) -> Void)
 
+    /// Optimistic updates the specified fields from an order.
+    ///
+    case optimisticUpdateOrder(siteID: Int64, order: Order, fields: [OrderUpdateField], onCompletion: (Result<Order, Error>) -> Void)
```

Comment:
I'm torn when it comes to naming this - I want to make it grammatically correct but it sounds weird to have an adverb in a method name. What do you think about `updateOrderOptimistically`?

Reply:
mmm... 🤔 Yep, I really like `updateOrderOptimistically` and I agree with you it sounds better 👍🏼 

Thanks!